### ISSUE TITLE
Can we get the time in seconds spent by users?

### ISSUE THREAD

**Issue by jihunim** [DeviceFarmer/stf#168]
**Is your feature request related to a problem? Please describe.**
I want to check how much users are using STF

**Describe the solution you'd like**
For example, put the following value to all `user` of the table `users`
`"timeUsed": 900`

**Comment by koral--** [comment]
What exactly do you mean by "spent" time?
The time of using the devices?

**Comment by jupe** [comment]
I guess @jihunim means the time of using the device (allocation time). That would be interested view. Also timeout for allocation would be useful feature.

**Comment by jay0129** [comment]
@koral-- Yes I meant the time of using the devices, sorry for the unclear comment.
This is because I want to know how long users are using the devices.

**Comment by jihunim** [comment]
@koral-- Yes I meant the time of using the devices, sorry for the unclear comment.

**Comment by sudhalucky** [comment]
I achieved this by adding a table called deviceUsage in tables.js and edited below methods in api.js

`dbapi.setDeviceOwner = function(serial, owner) {
  return dbapi.loadDeviceBySerial(serial).then(function(device) {
      return db.run(r.table('deviceUsage').insert({
        date: r.now().date(),
        location: device.provider.name,
        serial: device.serial,
        model: device.model,
        platform: device.platform,
        name: owner.name,
        email: owner.email,
        ownat: r.now().toEpochTime(),
        releaseat: false
      }
    , {
        durability: 'soft'
      }))
  })
  .then(function(stats) {
    return db.run(r.table('devices').get(serial).update({
      owner: owner
    }))
  })
}

dbapi.unsetDeviceOwner = function(serial, owner) {
  return db.run(r.table('usagelogs').filter({"serial":serial, "email":owner.email, "releaseat":false}).update({
    releaseat: r.now().toEpochTime()
  }))
  .then(function(stats) {
    return db.run(r.table('devices').get(serial).update({
      owner: null
    }))    
  })
}
`
